Library/OcCpuLib/OcCpuLib.c

```diff
@@ -953,16 +981,16 @@ OcCpuScanProcessor (
 
   DEBUG ((
     DEBUG_INFO,
-    "OCCPU: %a %a %11lld %5dMHz\n",
+    "OCCPU: %a %a %11,LuHz %5LuMHz\n",
```

Comment:
`%11,Lu` is that from that printf? Honestly, I do not see how it is supposed to work…

Reply:
It's not standard for `printf` but it is part of the [EDK2 format string spec for Print Lib](https://github.com/tianocore/edk2/blob/3806e1fd139775610d8f2e7541a916c3a91ad989/MdePkg/Include/Library/PrintLib.h#L39) you sent me:
```
- ,
  - Place a comma every 3rd digit of the number.  Only valid for type d.
    If 0 is also specified, then 0 is ignored.
```

I tested the output and it prints `3,000,000,000` instead of `3000000000`. Now, whether the `11` applies before or after the commas are added, you've got me. That I didn't check.

Reply:
Though reading this, perhaps it's supposed to be `%,14Lu` (to account for the commas) instead. Will change.

Reply:
They say it is only valid for d, and not u, but I guess they lie. To be honest, in Russia and Germany comma works as a floating point separator, and this change may make it quite confusing in both log and code reading.

If you think it is still worth trying to preserve it, could probably check how edk2 guys do it. I am away from keyboard at the moment, and github does not let me search commas.

Reply:
Eh, I'll just remove it. It's only a minor convenience to break up the characters. The docs don't say any of the flags work with `u`, only `d`. `u` goes completely unmentioned, including when discussing the `L` flag:
```
- L, l
  - The number being printed is size UINT64.  Only valid for types X, x, and d.
    If this flag is not specified, then the number being printed is size int.
```
I assume it's just a mistake in the docs but maybe it should be changed to `%11Ld` even.

Reply:
Just a mistake in the docs in my opinion. Should report that sooner or later I guess. Go with `%11Lu`.

Reply:
Fixed.